README.md

```diff
@@ -10,7 +10,7 @@ PackageCompiler is a Julia package with three main purposes:
 
   2. Creating "apps" which are a bundle of files including an executable that can be sent and run on other machines without Julia being installed on that machine.
 
-  3. Creating a relocatable C library bundle form of Julia code.
```

Comment:
Actually this seems correct.

Reply:
I still don't understand what it means but it must be me then :+1:

Reply:
At first I thought it was not clear either, but a second read it made sense. Maybe inversion helps clarity:

"Creating a relocatable bundle of Julia code in the form of a C library."